### PR TITLE
tests: organize behave steps

### PR DESCRIPTION
**Pull request by orndorffgrant**
## Note
This is only intended to organize our existing steps, not to change any of them.
After they're organized (after this is merged), we will be able to consolidate or remove unneeded steps or do other refactors more easily

## Test Steps

Integration tests should still pass.
Check my organization and tell me if it doesn't make sense or if something is mis-categorized or anything like that.

## Desired commit type

 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist

 - [ ] I have updated or added any unit tests accordingly
 - [x] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

File: features/steps/attach.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        ),
+        user_spec="as non-root",
+    )
+
+
+@when("I wait for the magic attach token to be activated")
+def when_i_wait_for_magic_attach_token(context):
+    when_i_run_command(
+        context=context,
+        command="pro api u.pro.attach.magic.wait.v1 --args magic_token={}".format(  # noqa
```
Comment (lucasmoura) [inline features/steps/attach.py:47]: Can we also isolate the magic-attach steps in a separate file ?

**Review by lucasmoura — COMMENTED** [review]
Great refactoring @orndorffgrant
Just a minor nit to also isolate the magic-attach logic

File: features/steps/attach.py (hunk 1 above)
Comment (orndorffgrant) [reply features/steps/attach.py:47]: Will do!

**Review by renanrodrigo — APPROVED** [review]
this is great

**Pull request merged**